core/src/test/scala/org/apache/spark/deploy/rest/SubmitRestProtocolSuite.scala

```diff
@@ -18,11 +18,9 @@
 package org.apache.spark.deploy.rest
 
 import java.lang.Boolean
-import java.lang.Integer
-
-import org.json4s.jackson.JsonMethods._
 
 import org.apache.spark.{SparkConf, SparkFunSuite}
+import org.json4s.jackson.JsonMethods._
```

Comment:
Please check Spark code style guide about how to organize imports: https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide